razorpay-payments.php

```diff
@@ -173,6 +226,8 @@ public function generate_razorpay_form($order_id)
 <form name='razorpayform' action="$redirect_url" method="POST">
     <input type="hidden" name="merchant_order_id" value="$order_id">
```

Comment:
I don't see `merchant_order_id` field being used anywhere after form submission either. Drop it?